features/run_tests.feature

```diff
@@ -0,0 +1,52 @@
+Feature: run tests
+
+  Background:
+    Given I run "mycha install"
+    And I enter "spec/spec_helper.coffee"
+    And I wait for the command to exit
+
+
+  Scenario: no tests
+    Given I have no test files
+    When I run "mycha"
+    Then I see the output "0 passing"
+
+
+  Scenario: passing test
+    Given I have the file "one_spec.coffee" which passes
+    When I run "mycha"
+    Then I see the output "1 passing"
+
+
+  Scenario: failing test
+    Given I have the file "one_spec.coffee" which fails
+    When I run "mycha"
+    Then I see the output "1 failing"
+    And it exits with non-zero status
+
+
+  Scenario: nested test
+    Given I have the file "dir/one_spec.coffee" which passes
+    When I run "mycha"
+    Then I see the output "1 passing"
+
+
+  Scenario: multiple tests
+    Given I have the file "one_spec.coffee" which passes
+    And I have the file "dir/two_spec.coffee" which passes
+    When I run "mycha"
+    Then I see the output "2 passing"
+
+
+  Scenario: running file
+    Given I have the file "one_spec.coffee" which passes
+    And I have the file "dir/two_spec.coffee" which passes
+    When I run "mycha one_spec.coffee"
+    Then I see the output "1 passing"
+
+
+  Scenario: running folder
+    Given I have the file "one_spec.coffee" which passes
```

Comment:
This file specifies 3 different things:
* the output for various passing and failing specs
* how spec files are found/globbed
* how individual spec files can be provided via the command line

I would separate those into separate files, and use scenario outlines there.